packages/formdescriptioneditors/frontend/sirius-components-formdescriptioneditors/src/Group.tsx

```diff
@@ -88,14 +89,18 @@ const useGroupEntryStyles = makeStyles<Theme, GroupStyleProps>((theme) => ({
   verticalSections: {
     display: 'flex',
     flexDirection: 'column',
-    alignItems: 'stretch',
     paddingLeft: theme.spacing(1),
     overflowX: 'auto',
+    gap: ({ borderStyle }) => (borderStyle ? borderStyle?.gap : null),
+    alignItems: ({ borderStyle }) => borderStyle?.alignItems || 'stretch',
+    justifyContent: ({ borderStyle }) => borderStyle?.justifyContent || null,
   },
   adaptableSections: {
     display: 'grid',
     gridTemplateColumns: 'repeat(auto-fit, minmax(250px, 1fr))',
-    gap: theme.spacing(2),
+    gap: ({ borderStyle }) => (borderStyle ? borderStyle?.gap : theme.spacing(2)),
+    alignItems: ({ borderStyle }) => borderStyle?.alignItems || null,
+    justifyContent: ({ borderStyle }) => borderStyle?.justifyContent || null,
```

Comment:
I somehow missed the fact that here the options are used to customize the grid algorithm but only half of it. We should either let specifiers specify and customize a Grid layout or not. This will not behave at all the way specifier may expect it.

Reply:
Customization is for flexbox only and the properties are not link to border anymore